descriptions/SDK.UnityURP.md

```diff
@@ -0,0 +1 @@
+[**URP**](https://docs.unity3d.com/Packages/com.unity.render-pipelines.universal@11.0/manual/index.html) The Universal Render Pipeline is a prebuilt Scriptable Render Pipeline, made by Unity. URP provides artist-friendly workflows that let you quickly and easily create optimized graphics across a range of platforms, from mobile to high-end consoles and PCs.
```

Comment:
Needs a little better wording otherwise URP is out of place: https://github.com/SteamDatabase/FileDetectionRuleSets/blob/349b13bb5674a8ce07f32af6c99cbae245ee6d40/descriptions/SDK.UnityURP.md

Reply:
> Needs a little better wording otherwise URP is out of place: https://github.com/SteamDatabase/FileDetectionRuleSets/blob/349b13bb5674a8ce07f32af6c99cbae245ee6d40/descriptions/SDK.UnityURP.md

1. URP Is a cross-platform Scriptable Render Pipeline built by Unity with the intent of a wide platform support range.
2. URP is a cross-platform Scriptable Render Pipeline built by Unity to target all modern platforms with a low performance overhead. 

And a few others. Not really sure what to choose. 
URP is similar to HDRP but with a focus on supporting all platforms (Mobile, Switch, PCs, Consoles) where HDRP is focused on extremely high graphics and platform support for powerful hardware only (Next-gen consoles, PCs). 

But URP is 'universal', in it that you can still achieve good looking visuals. Just not as good as HDRP, and harder without custom tools or assets. Where HDRP has it all provided by the engine.